conductor/lib/maya_utils.py

```diff
@@ -666,23 +655,82 @@ def parse_xgen_file(path, node):
 
     return file_paths
 
-#  Parse the OCIO config file to get the location of the associated LUT files
 
+def scrape_ocio_dependencies():
+    '''
+    Find the ocio config file (if it exists) and scrape it for external file dependencies
+    '''
+    config_filepath = get_ocio_config_filepath()
+    if config_filepath:
+        return parse_ocio_config_paths(config_filepath)
+    return []
 
-def parse_ocio_config(config_file):
 
-    def bunk_constructor(loader, node):
-        pass
+def get_ocio_config_filepath():
+    '''
+    Return the OCIO config filepath from maya settings.
 
-    yaml.add_constructor(u"ColorSpace", bunk_constructor)
-    yaml.add_constructor(u"View", bunk_constructor)
+    Only return the filepath if color managment is enabled and OCIO config is enabled.  
+    '''
+    if (cmds.colorManagementPrefs(q=True, cmEnabled=True) and
```

Comment:
The surrounding parenthesis aren't necessary.

Reply:
false